src/test_utils/katana/mod.rs

```diff
@@ -42,39 +49,86 @@ pub fn katana_config() -> StarknetConfig {
 }
 
 /// Returns a `TestSequencer` configured for Kakarot.
+#[cfg(any(test, feature = "arbitrary", feature = "testing"))]
 async fn katana_sequencer() -> TestSequencer {
     TestSequencer::start(SequencerConfig { no_mining: false, block_time: None, messaging: None }, katana_config()).await
 }
 
+/// Represents the Katana test environment.
 pub struct Katana {
+    /// The test sequencer instance for managing test execution.
     pub sequencer: TestSequencer,
+    /// The Kakarot EOA (Externally Owned Account) instance.
     pub eoa: KakarotEOA<Arc<JsonRpcClient<HttpTransport>>>,
+    /// Mock data stored in a HashMap, representing the database.
+    pub mock_data: HashMap<CollectionDB, Vec<StoredData>>,
+    /// The port number used for communication.
+    pub port: u16,
+    /// Option to store the Docker container instance.
+    /// It holds `Some` when the container is running, and `None` otherwise.
+    pub container: Option<Container<'static, GenericImage>>,
 }
 
-impl Katana {
-    pub async fn new() -> Self {
+impl<'a> Katana {
+    #[cfg(any(test, feature = "arbitrary", feature = "testing"))]
+    pub async fn new(rnd_bytes_size: usize) -> Self {
         let sequencer = katana_sequencer().await;
         let starknet_provider = Arc::new(JsonRpcClient::new(HttpTransport::new(sequencer.url())));
 
-        Self::initialize(sequencer, starknet_provider).await
+        Self::initialize(sequencer, starknet_provider, rnd_bytes_size).await
     }
 
     /// Initializes the Katana test environment.
-    async fn initialize(sequencer: TestSequencer, starknet_provider: Arc<JsonRpcClient<HttpTransport>>) -> Self {
-        // Load PK
+    #[cfg(any(test, feature = "arbitrary", feature = "testing"))]
+    async fn initialize(
+        sequencer: TestSequencer,
+        starknet_provider: Arc<JsonRpcClient<HttpTransport>>,
+        rnd_bytes_size: usize,
+    ) -> Self {
+        // Load the private key from the environment variables.
         dotenvy::dotenv().expect("Failed to load .env file");
         let pk = std::env::var("EVM_PRIVATE_KEY").expect("Failed to get EVM private key");
         let pk = B256::from_str(&pk).expect("Failed to parse EVM private key");
 
-        // Create a Kakarot client
-        let database = mock_database().await;
+        // Initialize a MongoFuzzer instance with the specified random bytes size.
+        let mut mongo_fuzzer = MongoFuzzer::new(rnd_bytes_size).await;
+        // Get the port number for communication.
+        let port = mongo_fuzzer.port();
+
+        // Run a Docker container with the MongoDB image.
+        let container = DOCKER_CLI.run(mongo_fuzzer.get_mongo_image());
```

Comment:
in `Rust`, we usually avoid doing getters with `get`
https://rust-lang.github.io/api-guidelines/naming.html#:~:text=Getter%20names%20follow%20Rust%20convention,for%20getters%20in%20Rust%20code.&text=The%20get%20naming%20is%20used,the%20content%20of%20a%20Cell%20.

Reply:
Yep, solved.